docs/adv-topics/uefi-secureboot.md

```diff
@@ -0,0 +1,131 @@
+
+!!! note
+    Raspberry Pi does not use or support UEFI or SecureBoot
+
+## Generating a Machine Owner Key
+Generation of a Machine Owner Key (MOK) is can be done with the
+`asl-setup-dkms-mok` utility. This tool appeared in the package
+asl3-3.3.0-1.
+
+1. Run the command `sudo asl-setup-dkms-mok`. This will prompt for 
+several items of information. An example run will look like:
+
+    ```
+    $ sudo asl-setup-dkms-mok
+    !!!
+    !!! WARNING - RUNNING THIS SCRIPT WILL REGENERATE THE
+    !!! DEFAULT MACHINE OWNER KEY (MOK) FOR SECURE BOOT. IN GENERAL YOU
+    !!! ARE RUNNING THIS SCRIPT BECAUSE YOU NEED TO AND YOU KNOW WHY
+    !!! YOU ARE DOING IT. HOWEVER BE WARNED!
+    !!!
+    ENTER "galaxy" TO CONTINUE>> galaxy
+
+    Generating an MOK requires a name for the certificate. This can be
+    anything, however your name or your callsign is recommended.
+
+    Name or callsign>> N8EI
+
+    Generating the MOK...
+    ......+...+...+.....+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*................+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.+.........+......+...+.+...........+...+.+...+..+.+...+..............+.......+...........+.........+................+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    .....+............+.........+.+.....+....+...+...+..............+.+..+...............+.+..+...+.+........+.+......+........+...............+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*.......+...+............+..+....+............+..+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++*....+.....+...+...........................+......+.......+...............+.....+......+..........+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+    -----
+
+    Enter a one-time password for this key. This password will be prompted
+    for at the next reboot to properly install the key in the UEFI
+    bootloader keyring. This does not need to be an incredibly complex
+    password and you may write it down until it's used.
+
+    input password:
+    input password again:
+
+    Setup /etc/dkms/framework.conf.d/asl.conf
+    Generate /etc/dkms/asl_sign_helper.sh
+
+    You must reboot the system and enter the OTP to load the key into SecureBoot
+    This will appear on your directly-connected monitor or your cloud console
+    ```
+
+2. Prepare to Reboot the System. Make sure a monitor is attached to the system or, if using
+a cloud virtual machine / VPS, make sure that a remote virtual
+console is connected. Then, reboot the system.
+
+    !!! warning "MOK Load Staging is One-Time Only"
+        Note that the MOK is only staged for load one time. If the MOK load prompt
+        is not responded to at the first/next reboot, the MOK staged for load
+        is wiped and Step #1 will have to be run again.
+
+3. Complete the steps to review, unlock, and load the MOK
+    !!! tip "The next step varies by vendor"
```

Comment:
Did you want this "tip" to be inline in the doc?  or somehow stand out?

Reply:
Oops. Needed an extra linebreak for that to render right.